app/Controller/TagsController.php

```diff
@@ -71,31 +74,31 @@ public function index($favouritesOnly = false)
         }
         if ($this->_isRest()) {
             unset($this->paginate['limit']);
-            unset($this->paginate['contain']['EventTag']);
```

Comment:
This will still mean that we load every event tag and every attribute tag. On instances with high tag volumes (think attributes in the 100M range, 7-8 tags per attribute) this will return a massive amount of data.

Reply:
Yes, I know. Now I am working how to fix it without loading all `EventTag` and `AttributeTag`.

Reply:
An option would be to loop through the results and run a count() on those two objects for each value. It would be slow in terms of querying the data, but at least we wouldn't run into crazy memory usage issues. I am still not convinced that having the accurate number here is really something worth the massive performance hit.